packages/react-scripts/scripts/start.js

```diff
@@ -42,6 +42,11 @@ if (!checkRequiredFiles([paths.appHtml, paths.appIndexJs])) {
   process.exit(1);
 }
 
+process.stdin.on('end', function() {
```

Comment:
If we break we'll learn from that. I think it's fine to start with on by default in the next major (technically minor since we're in 0.x), and see what breaks.